docs/core/project-sdk/msbuild-props.md

```diff
@@ -12,6 +12,123 @@ This page is a reference for the MSBuild properties and items that you can use t
 > [!NOTE]
 > This page is a work in progress and does not list all of the useful MSBuild properties for the .NET SDK. For a list of common MSBuild properties, see [Common MSBuild properties](/visualstudio/msbuild/common-msbuild-project-properties).
 
+## Assembly validation properties
+
+These properties and items are passed to the `ValidateAssemblies` task. For more information about assembly validation, see [Assembly validation](../../fundamentals/apicompat/assembly-validation.md).
+
+The following MSBuild properties and items apply to assembly validation:
+
+- [ApiCompatEnableRuleAttributesMustMatch](#apicompatenableruleattributesmustmatch)
+- [ApiCompatEnableRuleCannotChangeParameterName](#apicompatenablerulecannotchangeparametername)
+- [ApiCompatExcludeAttributesFile](#apicompatexcludeattributesfile)
+- [ApiCompatGenerateSuppressionFile](#apicompatgeneratesuppressionfile)
+- [ApiCompatLeftAssemblies](#apicompatleftassemblies)
+- [ApiCompatLeftAssembliesReferences](#apicompatleftassembliesreferences)
+- [ApiCompatLeftAssembliesTransformationPattern](#apicompatleftassembliestransformationpattern)
+- [ApiCompatRightAssemblies](#apicompatrightassemblies)
+- [ApiCompatRightAssembliesReferences](#apicompatrightassembliesreferences)
+- [ApiCompatRightAssembliesTransformationPattern](#apicompatrightassembliestransformationpattern)
+- [ApiCompatPermitUnnecessarySuppressions](#apicompatpermitunnecessarysuppressions)
+- [ApiCompatPreserveUnnecessarySuppressions](#apicompatpreserveunnecessarysuppressions)
+- [ApiCompatRespectInternals](#apicompatrespectinternals)
+- [ApiCompatStrictMode](#apicompatstrictmode)
+- [ApiCompatSuppressionFile](#apicompatsuppressionfile)
+- [ApiCompatSuppressionOutputFile](#apicompatsuppressionoutputfile)
+- [ApiCompatValidateAssemblies](#apicompatvalidateassemblies)
+- [ApiCompatValidateAssembliesSemaphoreFile](#apicompatvalidateassembliessemaphorefile)
+- [NoWarn](#nowarn)
+- [RoslynAssembliesPath](#roslynassembliespath)
+
+### ApiCompatLeftAssemblies
+
+The `ApiCompatLeftAssemblies` item specifies the path to one or more assemblies that serve as the *left side* to compare. This property is required when `ApiCompatValidateAssemblies` is set to `true`.
```

Comment:
> This property is required when `ApiCompatValidateAssemblies` is set to `true`.

If the intent here is to document the `ValidateAssembliesTask` then this makes sense. I don't expect many customers to invoke the task themselves. This is really just for advanced scenarios.

If customers want to compare assemblies (outside of packages) with APICompat, they just need to do the following:
1.  Package reference `Microsoft.DotNet.ApiCompat.Task`
2. Set `ApiCompatValidateAssemblies` to true

`ApiCompatLeftAssemblies` and `ApiCompatRightAssemblies` (including the assembly references) will be calculated automatically by the corresponding msbuild targets in the nuget package.

Note that while this feature is used by dotnet/runtime but it's more or less experimental. We haven't yet fine-tuned all the defaults. It might not work great outside of dotnet/runtime (which has reference assemblies that are passed in as the left).

It might be better to move the properties that only apply to this feature out of this file and into the dedicated assembly-validation.md.

Reply:
Okay, I think I'll just omit these properties for now since it's experimental.